docs/samples/Microsoft.ML.Samples/Dynamic/IidChangePointDetectorTransform.cs

```diff
@@ -1,13 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Interesting, good one, I wonder if we should have some mechanism to check for this sort of thing, right now it's purely manual.